Fundamentals/Setup/Server-Setup/azure-web-apps-v8.md

```diff
@@ -32,11 +32,15 @@ You need to add these `appSettings`:
     value="Examine.LuceneEngine.Directories.SyncTempEnvDirectoryFactory, Examine" />
 ```
 
+You should also add an appsetting `WEBSITE_DISABLE_OVERLAPPED_RECYCLING` with a value of 1 to your Azure WebApp (not this needs to be added to the App configuration via the Azure portal and not via Web.config. This setting prevents Umbraco from starting on an instance before a shutdown is complete - which is important as Umbraco in Azure needs to acquire an exclusive `MainDom` lock in order to start.
```

Comment:
This recommendation was removed for 8.6.3+ - not sure why. https://github.com/umbraco/UmbracoDocs/blob/main/Fundamentals/Setup/Server-Setup/Load-Balancing/azure-web-apps-v8.md#v860---v863

Reply:
@nul800sebastiaan I am pretty certain it needs to be there still. I have observed locking issues when trying to acquire the MainDom without this.